Sources/Interpreter/Memory.swift

```diff
@@ -357,3 +372,99 @@ public extension Memory.Address {
   static func -=(l: inout Self, r: Int)  { l = l - r }
 
 }
+
+extension Memory.Allocation {
+
+  /// Returns initaialized region containing offset `o`.
```

Comment:
```suggestion
  /// Returns the composed region containing offset `o`.
```

composed != initialized.  We decompose a thing before we deinitialize it and we compose it after initializing it.

I know it's subtle, but articles (“the,” “an,” etc.) really matter here.  If it was “an” instead of “the” it would imply we might be creating the composed region.

Should we have a precondition check that the result actually does contain `o`?